### PR TITLE
revised node-tab document

### PR DESCRIPTION
**Pull request by Ancairon**
##### Summary

I moved the old doc too, as it is best to have everything under one folder.

I addressed the issue that the doc was outdated, and I explained what is available, without giving too detailed info on _how_ to do it, because that might change and we should not be having to explain how it is done, it should be intuitive enough from the UI.

### CODE REVIEW

File: docs/dashboard/nodes-tab.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+# Nodes tab
+
+The nodes tab provides a summarized view of your [War Room](https://github.com/netdata/netdata/blob/master/docs/cloud/manage/organize-your-infrastrucutre-invite-your-team.md#netdata-cloud-war-rooms), allowing you to view quick information per node.
+
+> **Tip**  
+>
+> Keep in mind that all of the configurations mentioned below are persistent and visible across all users.
```
Comment (ilyam8) [inline docs/dashboard/nodes-tab.md:7]:
```suggestion
> Keep in mind that all configurations mentioned below are persistent and visible across all users.
```

File: docs/dashboard/nodes-tab.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+## Center information view
+
+The center information view consists of one row per node, and can be configured and filtered by the user.
+
+### Filtering and adjusting the view
+
+In the top right hand corner you can:
```
Comment (ilyam8) [inline docs/dashboard/nodes-tab.md:15]:
```suggestion
In the top right-hand corner, you can:
```

File: docs/dashboard/nodes-tab.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+- Order the nodes per status or per alert status
+- Select which charts you want to be displayed as quick reference points
+
+### Node row
+
+Each node row allows you to:
+
+- View the node's status
+- Go to a single node dashboard, by clicking the node name
+- View information about the node, along with a button to display more in the right hand side bar
```
Comment (ilyam8) [inline docs/dashboard/nodes-tab.md:26]:
```suggestion
- View information about the node, along with a button to display more in the right-hand sidebar
```

File: docs/dashboard/nodes-tab.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+- View active alerts for the node
+- View Machine Learning status
+- View Functions capability status
+- Add configuration (beta)
+- [Add alert silencing rules](https://github.com/netdata/netdata/blob/master/docs/cloud/alerts-notifications/manage-alert-notification-silencing-rules.md)
+- View a set of key metrics that can be configured
+
+## Right bar
+
+The bar on the right hand side provides additional information about the nodes in the War Room and also allows to filter what is displayed in the [center information view](#center-information-view).
```
Comment (ilyam8) [inline docs/dashboard/nodes-tab.md:36]:
```suggestion
The bar on the right-hand side provides additional information about the nodes in the War Room and allows you to filter what is displayed in the [center information view](#center-information-view).
```

File: docs/dashboard/nodes-tab.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+The second tab allows you to filter which nodes are displayed, you can filter by:
+
+- Host labels
+- Node status
+- Netdata version
+- Individual nodes
+
+### Alerts sub-tab
+
+The third tab presents the room alerts, and allows you to see more info about each alert.
```
Comment (ilyam8) [inline docs/dashboard/nodes-tab.md:53]:
```suggestion
The third tab displays room alerts and allows you to see additional information about each alert.
```

File: docs/dashboard/nodes-tab.md [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+
+### Info sub-tab
+
+The last tab presents information about a node, by clicking the `i` icon from a node's row, right next to it's name.
```
Comment (ilyam8) [inline docs/dashboard/nodes-tab.md:57]:
```suggestion
The last tab presents information about a node, by clicking the `i` icon from a node's row, right next to its name.
```

File: docs/dashboard/nodes-tab.md [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+
+- View the node's status
+- Go to a single node dashboard, by clicking the node name
+- View information about the node, along with a button to display more in the right-hand sidebar
+- View active alerts for the node
+- View Machine Learning status
+- View Functions capability status
+- Add configuration (beta)
+- [Add alert silencing rules](https://github.com/netdata/netdata/blob/master/docs/cloud/alerts-notifications/manage-alert-notification-silencing-rules.md)
+- View a set of key metrics that can be configured
```
Comment (hugovalente-pm) [inline docs/dashboard/nodes-tab.md:32]:
```suggestion
- View a set of key attributes collected on your node
```

I'm guessing you're referring to the node attributes like OS, CPU, RAM, etc.

**Review by ilyam8 — APPROVED** [review]

**Pull request merged**